src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -20,6 +20,7 @@ public class AzureAppConfigurationOptions
     {
         private const int MaxRetries = 2;
         private static readonly TimeSpan MaxRetryDelay = TimeSpan.FromMinutes(1);
+        private readonly string[] configStoreNameSplitSeparator = { ConfigurationStoreConstants.ConfigStoreNameReplicaSeparator };
```

Comment:
>configStoreNameSplitSeparator [](http://example.com/codeflow?start=34&length=29)

Nit: ConfigStoreNameReplicaSeparator - to keep the same name as the constant.
Also, it can be `static readonly` and the naming convention we use for members like this is PascalCase.

Reply:
With the new implementation, we don't need this constant anymore.